xla/debug_options_flags.cc

```diff
@@ -834,6 +839,24 @@ void MakeDebugOptionsFlags(std::vector<tsl::Flag>* flag_list,
       bool_setter_for(&DebugOptions::set_xla_gpu_enable_async_all_to_all),
       debug_options->xla_gpu_enable_async_all_to_all(),
       "Converts synchronous all-to-all ops into asynchronous."));
+  flag_list->push_back(tsl::Flag(
+      "xla_gpu_enable_prescheduling_combiners",
+      bool_setter_for(
+          &DebugOptions::set_xla_gpu_enable_prescheduling_combiners),
+      debug_options->xla_gpu_enable_prescheduling_combiners(),
+      "Enables the running of collective combiners before scheduling."));
+  flag_list->push_back(tsl::Flag(
+      "xla_gpu_enable_postscheduling_combiners",
+      bool_setter_for(
+          &DebugOptions::set_xla_gpu_enable_postscheduling_combiners),
+      debug_options->xla_gpu_enable_postscheduling_combiners(),
+      "Enables the running of collective combiners after scheduling."));
+  flag_list->push_back(tsl::Flag(
+      "xla_gpu_postscheduling_all_reduce_strategy",
+      string_setter_for(
+          &DebugOptions::set_xla_gpu_postscheduling_all_reduce_strategy),
+      debug_options->xla_gpu_postscheduling_all_reduce_strategy(),
+      "Sets the strategy for the post-scheduling all-reduce combiner."));
```

Comment:
Can you list the valid option and a quick definition of them?
This make it much easier to understand how to use it without digging in the code.